backend/app/expenses/service.py

```
@@ -341,21 +353,26 @@ async def update_expense(self, group_id: str, expense_id: str, updates: ExpenseU
                     if updated_expense:
                         # Create new settlements
                         await self._create_settlements_for_expense(updated_expense, user_id)
-                except Exception as e:
-                    logger.error(f"Warning: Failed to recalculate settlements: {e}",exc_info=True)
+                except Exception:
+                    logger.error(f"Warning: Failed to recalculate settlements",exc_info=True)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)